sickbeard/generic_queue.py

```diff
@@ -41,6 +41,8 @@ def __init__(self):
 
         self.lock = threading.Lock()
 
+        self.amActive = None
+
```

Comment:
Actually should be False (None is ok because we just check if blah.amActive)

self.amActive = True is missing from start of the run method.